totes

```diff
@@ -119,6 +119,17 @@ handle_user_input() {
     n | N)
       totes "$@" # propagate arguments
       ;;
+    e | E)
+      if [[ -z "$EDITOR" ]]; then
+        printf "%s\n" "No value for \$EDITOR is set, so unable to edit the requested file."
+        exit 0
+      fi
+      "$EDITOR" "$note_to_render"
+      # Date format: [[CC]YY]MMDDhhmm[.ss]
+      # Setting to 1:01 on jan 1, 2001
+      touch -t 0101010101 "$note_to_render"
```

Comment:
appreciate the `0101010101` timestamp 😄 Might be worth adding a comment mentioning that this puts it at the front of the rotation (could be deduced but I was confused for a sec)

Reply:
will do